rstudio/c9s-python-3.11/setup_rstudio.py

```diff
@@ -60,7 +84,7 @@ def _get_cmd(port):
         '--server-working-dir=' + os.getenv('HOME'),
         '--auth-none=1',
         '--www-frame-origin=same',
-        #'--www-address=0.0.0.0',
+        '--www-address='+ detect_env(),
```

Comment:
Right, the default value won't cut it

https://github.com/rstudio/rstudio/blob/56d05a406e78afdc7c3859697f71947b8da9f514/src/cpp/server/server-options.json#L130